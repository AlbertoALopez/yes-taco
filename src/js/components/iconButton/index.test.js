import React from 'react';
import IconButton from './index.js';

test('Button renders correctly', () => {
  const wrapper = shallow(<IconButton type="check" size={30} />);

  expect(wrapper).toMatchSnapshot();
});

test('Button has a click workking click handler', () => {
  const clickHandler = jest.fn();
  const wrapper = mount(<IconButton clickHandler={clickHandler} />);

  wrapper.find('Button').simulate('click');
  expect(clickHandler.mock.calls.length).toBe(1);
});
