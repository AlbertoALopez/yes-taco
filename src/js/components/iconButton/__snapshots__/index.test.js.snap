// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<MorphIcon
  color="#000"
  damping={15}
  size={30}
  stiffness={150}
  thickness={1}
  type="check"
/>
`;
